source/Makefile

```diff
@@ -133,11 +133,9 @@ OPTIM=-Os -flto -finline-small-functions -findirect-inlining -fdiagnostics-color
 GLOBAL_DEFINES += $(DEV_GLOBAL_DEFS) -D USE_RTOS_SYSTICK -D MODEL_$(model) -D VECT_TAB_OFFSET=$(bootldr_size)U
 
 # Without debug code
-DEBUG=
+DEBUG=-g

```

Comment:
-ggdb3 please, it allows GDB to expand macros

Reply:
Done. (`-g3` should do the same)